typescript/sdk/src/token/EvmERC20WarpRouteReader.ts

```diff
@@ -206,13 +213,27 @@ export class EvmERC20WarpRouteReader extends EvmRouterReader {
           error,
         );
       }
+      const hasTokenFeeInterface =
+        compareVersions(
+          tokenConfig.contractVersion!,
+          TOKEN_FEE_CONTRACT_VERSION,
+        ) >= 0;
```

Comment:
What do you think of moving this check into the `fetchTokenFee` method and here just assign the result of the method call to the `tokenFee` so you cna avoid the ternary below?

Reply:
done